### PR TITLE
Bump RabbitMQ.Client to the latest version

### PR DESCRIPTION
**Pull request by SeanFeldman**
PR updates the deployed package with the latest version of RabbitMQ client.

### CODE REVIEW

File: src/NServiceBus.Transport.RabbitMQ/NServiceBus.Transport.RabbitMQ.csproj [hunk 1]
```diff
@@ -12,12 +12,12 @@
...
+    <PackageReference Include="RabbitMQ.Client" Version="[5.1.2, 6.0.0)" />
   </ItemGroup>
 
   <PropertyGroup>
     <PackageId>NServiceBus.RabbitMQ</PackageId>
     <Description>RabbitMQ support for NServiceBus</Description>
   </PropertyGroup>
 
-</Project>
+</Project>
```
Comment (bording) [inline src/NServiceBus.Transport.RabbitMQ/NServiceBus.Transport.RabbitMQ.csproj:23]: nitpick: Let's add the newline back.

Comment (SeanFeldman) [reply src/NServiceBus.Transport.RabbitMQ/NServiceBus.Transport.RabbitMQ.csproj:23]: What's the purpose of the blank new line?

Comment (bording) [reply src/NServiceBus.Transport.RabbitMQ/NServiceBus.Transport.RabbitMQ.csproj:23]: Its the standard way that project files are defined in the template, so let's keep it consistent.

Comment (SeanFeldman) [reply src/NServiceBus.Transport.RabbitMQ/NServiceBus.Transport.RabbitMQ.csproj:23]: > Its the standard way that project files are defined in the template, so let's keep it consistent.

Could you please point me to the standard to see it? I am not aware of this requirement. While it's a nitpick, if this is something we're going to apply to all the projects, I'd rather do it now, understanding the rationale.

Comment (SeanFeldman) [reply src/NServiceBus.Transport.RabbitMQ/NServiceBus.Transport.RabbitMQ.csproj:23]: Discussed and it's not needed.

Comment (bording) [reply src/NServiceBus.Transport.RabbitMQ/NServiceBus.Transport.RabbitMQ.csproj:23]: I can see that it's already inconsistent in the repo, so I guess it doesn't matter.

**Pull request merged**